### DISCUSSION TITLE
What distro are you using?

### DISCUSSION THREAD

**Discussion by Justme488** [Justme488/game-converter#5] [General]
What distro are you using, and did everything work correctly?

**Comment by Codythedragonrude** [comment]
Void Linux 
I used xdeb to convert the package into an xbps formatted packaged 
Everything works fine as of march 2021

Keep up the good work this makes chd conversion so much easier thank you for making it

**Comment by Codythedragonrude** [comment]
Tested on Manjaro XFCE 21.0.2
Works Perfectly
Used debtap from aur to convert deb to arch package then had to install ciso from aur (this ciso dependency can be removed from the arch pkgbuild during debtap process if not needed)

**Comment by LuizSerapiao** [comment]
Tested on Zorin OS 16
Works like a charm, that makes compressing my roms a lot more convenient, thank you.

**Comment by Loughty** [comment]
Tested on Ubuntu 24, still works on 2025!

![Captura desde 2025-01-27 11-44-38](https://github.com/user-attachments/assets/46857e6a-004a-4971-8727-9ccb9d5d6926)

![Captura desde 2025-01-27 12-02-05](https://github.com/user-attachments/assets/f2928aae-6980-4cde-8b8d-d8c073bfcc28)

**Comment by fatalSCR1PT** [comment]
Tested on Arch Linux. 

Converted with Debtap from .DEB to PKG. Manually installed CISO as dependency. Then "sudo pacman -U [file path]" to install. 

I like to do this tasks with a UI. Thanks :)
